src/objects/tabl/zcl_abapgit_object_tabl_ddl.clas.abap

```diff
@@ -760,7 +761,14 @@ CLASS ZCL_ABAPGIT_OBJECT_TABL_DDL IMPLEMENTATION.
       rv_ddl = rv_ddl && |@AbapCatalog.dataMaintenance : \n|.
     ENDIF.
 
-    IF is_data-dd02v-pk_is_invhash = abap_true.
+    " doesn't exist on NW <= 750
+    ASSIGN
+      COMPONENT 'PK_IS_INVHASH'
+      OF STRUCTURE is_data-dd02v
+      TO <lv_pk_is_invhash>.
+    ASSERT sy-subrc = 0.
+
+    IF <lv_pk_is_invhash> = abap_true.
       rv_ddl = rv_ddl && |@AbapCatalog.primaryKey.invertedHashIndex : true\n|.
     ENDIF.
```

Comment:
this will dump in low versions? suggest

```suggestion
    IF sy-subrc = 0 AND <lv_pk_is_invhash> = abap_true.
      rv_ddl = rv_ddl && |@AbapCatalog.primaryKey.invertedHashIndex : true\n|.
    ENDIF.
```

Reply:
It will. I assumed this is something not available in these systems anyway?

Reply:
Btw. where is it used? Cannot find static references besides local test classes.